test/Reentrancy/ReadOnlyReentrancy/README.md

```diff
@@ -11,7 +11,7 @@
 
 
 ## Detailed Description
-So far, no attacks that follow this path were identified. However, it is worth exploring this example for educational purposes.
+Several recent attacks followed this novel pattern. In here, a theoretical example is provided to for educational purposes.
```

Comment:
FWIW, you can link our issue https://github.com/coinspect/learn-evm-attacks/issues/43 on this matter which documents multiple conducted attacks.

Reply:
@pcaversaccio I think it's a good idea. @MatthiasEgli are you interested in improving it by adding the issue to the "Sources and references" section?

Reply:
Done, added a link to the issue to the reference section.